serve-web/src/Controller/ApiController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use GuzzleHttp\ClientInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class ApiController extends AbstractController
+{
+    private ClientInterface $httpClient;
+
+    public function __construct(ClientInterface $httpClient, $apiEndpoint)
```

Comment:
Think this should be:

```suggestion
    public function __construct(ClientInterface $httpClient, string $apiEndpoint)
```